packages/block-library/src/file/index.js

```diff
@@ -203,7 +203,7 @@ export const settings = {
 
 		return ( href &&
 			<div>
-				{ ! isEmpty( fileName ) &&
+				{ ! RichText.isEmpty( fileName ) &&
```

Comment:
`fileName` is a string. Why should we need utility.

Reply:
Yes, we could get rid of the is empty utility as well now. I'm fine either way.

Reply:
Considering them as strings, I see no need for the utility.